test/cmdlineTests/ir_compiler_subobjects/output

```diff
@@ -48,38 +53,56 @@ object "D_16" {
     object "D_16_deployed" {
         code {
             {
+                /// @src 0:96:165
                 mstore(64, 128)
                 if iszero(lt(calldatasize(), 4))
                 {
                     let _1 := 0
-                    if eq(0x26121ff0, shr(224, calldataload(_1)))
```

Comment:
This whole code is essentially jumping back and forth between 96, 143 and 149. Is that an artefact of the optimizer (because of constants)?